Sources/TelegramVaporBot/Bot/Handlers/CommandHandler.swift

```diff
@@ -84,9 +84,9 @@ public class TGCommandHandler: TGHandlerPrtcl {
         return !commands.intersection(types).isEmpty
     }
     
-    public func handle(update: TGUpdate, bot: TGBotPrtcl) {
+    public func handle(update: TGUpdate, bot: TGBotPrtcl) async {
```

Comment:
it would be nice to have both usual and async implementation but not only an async one.